x-pack/plugins/synthetics/public/legacy_uptime/pages/overview.test.tsx

```diff
@@ -8,15 +8,8 @@
 import React from 'react';
 import { OverviewPageComponent } from './overview';
 import { render } from '../lib/helper/rtl_helpers';
-import { unifiedSearchPluginMock } from '@kbn/unified-search-plugin/public/mocks';
-import { setAutocomplete } from '@kbn/unified-search-plugin/public/services';
 
 describe('MonitorPage', () => {
-  beforeEach(() => {
-    const autocompleteStart = unifiedSearchPluginMock.createStartContract();
-    setAutocomplete(autocompleteStart.autocomplete);
-  });
-
```

Comment:
we added this because earlier our test was flaky, saying auto complete wasn't set.

Reply:
In this PR, we have removed the autocomplete setter and getter. And replaced it with getting autocomplete from Kibana service
```
this.services.unifiedSearch.autocomplete.getValueSuggestions
```

Reply:
do we need to do that in above test, or it's not required?

Reply:
I assume this is no longer required.

Reply:
I think this is no longer required.